tsconfig.json

```diff
@@ -26,6 +26,12 @@
       "@/*": ["./*"],
     },
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx",
+    ".next/types/**/*.ts",
+    "env.mjs",
```

Comment:
This had been removed. Don't add it back (happened due to bad resolution of merge conflicts during rebase)
```suggestion
```